addons/survey/tests/test_survey_performance.py

```diff
@@ -16,7 +16,7 @@ def test_survey_results_with_multiple_filters_mixed_model(self):
         """
         url = f'/survey/results/{self.survey.id}?filters=A,0,{self.gras_id}|L,0,{self.answer_pauline.id}'
         self.authenticate('survey_manager', 'survey_manager')
-        with self.assertQueryCount(default=31):
```

Comment:
It's linked to [errors](https://runbot.odoo.com/runbot/build/50963201) reported by the **runbot**, probably because of the logic I'd added in the **controller/mode**l. But this code has since been removed. I'm putting the test back to its original state to see.